network/monuments.csv

```diff
@@ -3,8 +3,8 @@ Mark,Domes Number,Mark Type,Type,Ground Relationship,Foundation Type,Foundation
 2406,,Forced Centering,Pillar,-1.66,Reinforced Concrete,2,2012-01-31T00:00:00Z,9999-01-01T00:00:00Z,,
 AHTI,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.5,Stainless Steel Rods,10,2009-01-01T00:00:00Z,9999-01-01T00:00:00Z,Sandstone,Site On Flat Linear Ridge Line With Steepening Slopes To The North And South.
 AKTO,,Forced Centering,Wyatt/Agnew Drilled-Braced,-1.5,Stainless Steel Rods,10,2006-04-27T00:00:00Z,9999-01-01T00:00:00Z,Sandstone,
-ALS1,,Forced Centering,Steel Mast,-2.1,Stainless Steel Rods,4,2023-02-24T01:08:38Z,9999-01-01T00:00:00Z,,
-ALS2,,Forced Centering,Steel Mast,-0.5,Stainless Steel Rods,1.5,2023-02-23T01:17:02Z,9999-01-01T00:00:00Z,,
+ALS1,,Forced Centering,Steel Mast,-2.1,Stainless Steel Rods,4,2023-02-24T01:08:38Z,2024-02-25T13:59:30Z,,
+ALS2,,Forced Centering,Steel Mast,-0.5,Stainless Steel Rods,1.5,2023-02-23T01:17:02Z,2024-03-13T20:59:30Z,,
```

Comment:
@EvanCGNS Could you please remove hours, minutes and seconds from the `Start` and `End` timestamps?